openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -136,7 +132,7 @@ def make_message_type(self, day_offset):
         raise NotImplementedError
 
 
-@task(base=LoggedTask, ignore_result=True, routing_key=ROUTING_KEY)
```

Comment:
I think we should only change line 43. These tasks down here should never be called by studio.

Reply:
@mulby I have updated the code.